### ISSUE TITLE
Update work progress

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#31]
## User story
1. As SDs
2. We need to document our work with sufficient information. 
3. So that the POs and industry partners can easily understand our progress and provide frequent feedback.

## Acceptance criteria
* Documentations from all SDs are available on sick extranet before deadline (14/05/2022)
* In case of multiple collaborators assigned on one task, an update from one SD is sufficient 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
